README.md

```diff
@@ -700,6 +702,44 @@ In this example, all saves to `MyTable` will raise an `Dynamoid::Errors::StaleOb
 
 Calls to `update` and `update!` also increment the `lock_version`, however they do not check the existing value. This guarantees that a update operation will raise an exception in a concurrent save operation, however a save operation will never cause an update to fail. Thus, `update` is useful & safe only for doing atomic operations (e.g. increment a value, add/remove from a set, etc), but should not be used in a read-modify-write pattern.
 
+
+### Backoff strategies
+
+
+You can use several methods what do efficiently in batch mode like `.find_all` and `.import`.
+
+By some reasons, for instanse table throughput limit is exceeded, some items could be not processed.
+By default operations will be re-run to process this items.
+
+Recommended way to handle throughput limits exceeding and throttling on the table is to use exponential backoff.
+
+There are two default backoff strategies - constant delay and truncated binary exponential backoff.
+By default no backof is used but you can specify one of the default ones:
```

Comment:
Change to:

> The default configuration is to not use backoff, but instead to *raise an error* when items fail to get processed.
> If you want to enable backoff, customize one of the following to meet your needs:

Reply:
@pboling I implemented what was proposed in the issue (https://github.com/Dynamoid/Dynamoid/issues/226#issuecomment-369736839)

> I would like both batch_get_item and batch_write_item to handle unprocessed items themselves and try to read/write unprocessed items until the end. It's a naive approach but it works and it hides problem from a client.